packages/sx05re/emuelec/bin/emuelec-utils

```diff
@@ -6,6 +6,47 @@
 # Source predefined functions and variables
 . /etc/profile
 
+function killall_midi()
+{
+	pkill -9 timidity
+  pkill -9 mt32d
+  pkill -9 fluidsynth
+	
+}
+
+function set_midi_source()
+{
+  local midi_source=$1
+  local midi_emulator=$2
+
+  killall_midi
+
+  [[ "${midi_emulator}" == "retrorun" ]] && return
+
+  local midi_output=OFF
+  
+  case ${midi_source} in
+    fluidsynth)
+      fluidsynth -is /storage/roms/bios/sc/SC-55.sf2 &
+      sleep 3
+      =$(cat /proc/asound/seq/clients | grep "Port   0 : \"S" | cut -d'"' -f2)
+      [[ -z "${MIDI_OUTPUT}" ]] && midi_output=OFF
+      ;;
+    timidity)
+      timidity -iA --module 1 &
+      midi_output="TiMidity port 0"
+			;;
+    mt32d)
+      mt32d -o 1 -i 24 -f /roms/bios &
```

Comment:
Here I was using -o 1 to use CM32-L roms that it is the mt32d latest device that was released. I think we should add and extra entry in emulationstation to select between mt32d original version and CM32-L as there are games that were programmed to use the original one. Original version flag is -o 2 . Let's discuss on discord @Langerz82  :)